src/current/_includes/common/licensing-faqs.md

```diff
@@ -0,0 +1,153 @@
+CockroachDB is made available under the [CockroachDB Software License][csl].
+
+## Types of licenses
+
+The table of licenses below refers to options for {{ site.data.products.core }} deployments. All {{ site.data.products.cloud }} deployments automatically have a valid **Enterprise** license.
+
+Type | Description
```

Comment:
We left out any mention of evaluation licenses. Was that intentional?

Reply:
what are "evaluation licenses" - is that something different than the self-serve Enterprise Trial license?  perhaps something that is handed out directly by sales?   I'm not seeing the text "evaluation" or "evaluation license" in the product brief

Reply:
The evaluation license type existed previously (see 1.1.3 in the product brief). My understanding is that they are used for POCs.

Reply:
thanks @spilchen for the catch, i just reviewed that list again, it seems we are missing multiple items from that list, my bad

@jaiayu do you want me to replicate here all of the license types from the product brief?  specifically, it looks like we're missing:

- Evaluation
- Development
- Pre-production

Or is it our preference that those license types are described and managed via interactions with a Sales process vs. being described in the public user documentation? 

I'm asking because up until now we have not listed out such licenses - [current FAQ](https://www.cockroachlabs.com/docs/stable/licensing-faqs#types-of-licenses) says there is:

- Apache
- BSL
- CCL

And doesn't talk about dev, eval etc licenses.

Presumably dev, eval, etc. licenses also existed under the previous regime, but we never documented them before

Happy to do it now if you want, please let me know

Reply:
@rmloveland I think it might be a good to mention that there is an option to evaluate CockroachDB without the telemetry requirements by contacting Sales. 

I'm not sure if we should list out the "Evaluation" license key, or if we should just say "Contact Sales if you want to try CockroachDB without telemetry requirements or if you require an extended trial period."

Reply:
Updated to add the following in a commit `Contact sales for longer/no-telemetry trials`

> <a href="mailto:sales@cockroachlabs.com">Contact Sales</a> if you want to try CockroachDB without telemetry requirements or if you require an extended trial period.